src/olympia/addons/tests/test_models.py

```diff
@@ -338,7 +334,7 @@ def test_filter_for_many_to_many(self):
         assert collection.addons.count() == 0
 
         # Only deleted.
-        self.addon.update(is_listed=True)
+        self.make_addon_unlisted(self.addon, listed=True)
```

Comment:
I think it's worth it. From a quick grep I didn't find any use of that argument in master currently, so it's limited to the changes in that PR anyway.